assign0/README.md

```diff
@@ -0,0 +1,354 @@
+Can you make the deposit? <b><i>yes</i></b>
+Deposit successful! You have 5532 Dogecoin remaining in your account.
+
+What is your name, then, traveller? <b><i>Michael</i></b>
+What is your passphrase? <b><i>parthsarin12345</i></b>
+Welcome through the Doors of Destiny! And it's been a pleasure doing business with you.
+
+(cs41-env)$ python chatbot.py
+Halt! Welcome to the Doors of Destiny. 
+Should you wish to proceed, you must identify yourself within the Book of Records. 
+
+Is your name present in our book? <b><i>yes</i></b>
+What is your name, traveller? <b><i>Michael</i></b>
+What is your passphrase? <b><i>cs41isacoolclass</i></b>
+The passphrase you presented does not match our records! Guards - arrest this intruder!
+</code></pre>
+
+This chatbot is adorable and geeky! Feel free to bring your personality and passion to this ☺️
+
+### Example 2: Simple Schedule
+
+Another example might be a Virtual Assistant chatbot, which allows users to schedule events (in which they would enter a time range, and a name for the event, which would be stored into a text file), and check whether an event was taking place at a given time. A sample run for such a system might appear as follows. (Note that this uses MMDDYYYY as an encoding scheme for dates and hours are represented as floating point numbers, so 14.5 means 2:30PM). Once again, user input is ***bolded and italicized***.
+
+<pre><code>
+(cs41-env)$ python chatbot.py
+Hello there, it's Hal, your friendly scheduling assistant! 
+
+Would you like to add a new event, or check an existing time slot? <b><i>add</i></b>
+What is the name of the event? <b><i>CS41 Lecture</i></b>
+On which day would you like to schedule the event? <b><i>03302021</i></b>
+What is the start time? <b><i>14.5</i></b>
+What is the end time? <b><i>16</i></b>
+
+Successfully added the event to your day!
+
+(cs41-env)$ python chatbot.py
+Hello there, it's Hal, your friendly scheduling assistant! 
+
+Would you like to add a new event, or check an existing time slot? <b><i>check</i></b>
+On which day would you like to check for scheduled events? <b><i>03302021</i></b>
+What time would you like to check for availability? <b><i>15</i></b>
+At that time, you'll be busy with CS41 Lecture.
+
+
+(cs41-env)$ python chatbot.py
+Hello there, it's Hal, your friendly scheduling assistant! 
+
+Would you like to add a new event, or check an existing time slot? <b><i>open the pod bay doors</i></b>
+I'm sorry Dave, I'm afraid I can't let you do that.
+</code></pre>
+
+### Example: File I/O
+Note that with both of these examples, we've left the scheme you use to store data to the file open-ended. For example, if designing an authentication system as in the first example, you might choose to create a text file with the following data format:
+
+```
+dogecoin_in_account:5632
+
+coopermj
+parthsarin12345
+
+psarin
+pythoniscool!!
+
+antonioferriss
+cs41isafunclass
+```
```

Comment:
@cooper-mj See my earlier comment about the difficulty of this formatting. We might want to have two files? One for the amount of dogecoin and another for the username/password combinations?

Also, keep in mind that people might not know what dogecoin is. Could we just gesture at "The keeper uses dogecoin, a type of currency"? (Nothing too distracting)

Reply:
Addressed/discussed in my comment above. And good suggestion with the Dogecoin, I'll edit that as well to make it clearer. Thanks for the awesome feedback on this document!